frontend/packages/ceph-storage-plugin/src/components/ocs-install/attached-devices/install-lso-sc.tsx

```diff
@@ -162,7 +170,15 @@ export const CreateOCS = withHandlePromise<CreateOCSProps & HandlePromiseProps>(
             <SelectNodesSection
               table={AttachedDevicesNodeTable}
               customData={{ filteredNodes, nodes, setNodes }}
-            />
+            >
+              {isMinimal && (
+                <div className="ceph-ocs-install__minimal-msg">
+                  Since the selected nodes do not satisfy the recommended requirements of 16 CPUs
```

Comment:
In BM/attached devices ocs install flow, we don't allow user to select nodes. `selected nodes` here doesn't seem correct.

Reply:
The selected nodes are coming from Storage CLass right?

Reply:
Yes, the nodes associated with the storage class are listed on the install page. My point is, UI doesn't allow selection of nodes on install page, so not sure whether naming it as selected nodes here is correct.

Reply:
We can sync up with @yuvalgalanti on coming monday. For now lets go with this text, if required, will take this as a bug fix.